ledger-core/virtual/integration/utils/server.go

```diff
@@ -188,6 +188,7 @@ func newServerExt(ctx context.Context, t Tester, errorFilterFn logcommon.ErrorFi
 	virtualDispatcher.CycleFn = s.onConveyorCycle
 	virtualDispatcher.EventlessSleep = -1 // disable EventlessSleep for proper WaitActiveThenIdleConveyor behavior
 	virtualDispatcher.MachineLogger = machineLogger
+	virtualDispatcher.MaxRunners = 10
```

Comment:
better to make it consistent with other tests please - hence use 4, not 10

Reply:
done